### PR TITLE
Add custom chain analytics event

### PR DESCRIPTION
**Pull request by 0xDaedalus**
### To Test
- [ ] Add a custom chain to the wallet
- [ ] Observe the `Custom Chain Added` event in posthog.

Latest build: [extension-builds-3236](https://github.com/tahowallet/extension/suites/12046398882/artifacts/633415473) (as of Wed, 05 Apr 2023 13:17:48 GMT).

### CODE REVIEW

File: background/main.ts [hunk 1]
```diff
@@ -1735,11 +1759,20 @@ export default class Main extends BaseService<never> {
...
+        const networkNameAtClose =
+          this.store.getState().ui.selectedAccount.network.name
         this.analyticsService.sendAnalyticsEvent(AnalyticsEvent.UI_SHOWN, {
           openTime: new Date(openTime).toISOString(),
           closeTime: new Date().toISOString(),
           openLength: (Date.now() - openTime) / 1e3,
+          networkName:
+            originalNetworkName === networkNameAtClose
+              ? originalNetworkName
+              : "switched networks",
```
Comment (hyphenized) [inline background/main.ts:1775]: This'll tell us users switched chains after the popup closed, but it won't catch network changes while the popup is open 🤔

**Review by hyphenized — DISMISSED** [review]
Looks good, left one suggestion

File: background/main.ts (hunk 1 above)
Comment (0xDaedalus) [reply background/main.ts:1775]: This should be fine - we don't need precise data about what chains our users are using - a directional idea is more than enough.

**Review by kkosiorowska — DISMISSED** [review]
I have tested the solution works great. ✅ Let's fix the conflicts and merge it.

**Review by kkosiorowska — APPROVED** [review]

**Pull request merged**